src/transports/tcp.rs

```diff
@@ -437,8 +439,7 @@ impl<Id: PeerId> Transport<Id> for TcpTransport<Id> {
                 .error("recv len", Some(format!("{:?}", err)))
         })?);
 
-        if res_size as usize > config.max_message_size_read {
-            dbg!("return error");
```

Comment:
And this

Reply:
that was the previous problem with usize. Using u32 now